commands/sandbox.go

```diff
@@ -34,12 +34,21 @@ import (
 )
 
 const (
-	nodeVersion       = "v16.13.0"
-	minSandboxVersion = "2.3.10-1.1.0"
+	// Minimum required version of the sandbox plugin code.  The first part is
+	// the version of the incorporated Nimbella CLI and the second part is the
+	// version of the bridge code in the sandbox plugin repository.
+	minSandboxVersion = "3.0.0-1.1.0"
```

Comment:
The version needed to be upgraded anyway, but this change is also important for its side-effect of forcing legacy users to run an upgrade once they download "this" version of doctl.   The upgrade step includes the renaming from ".nimbella" to a conforming name for credential storage.